nucliadb_sdk/nucliadb_sdk/search.py

```diff
@@ -48,47 +53,68 @@ def __init__(
         self.inner_search_results = inner_search_results
         self.client = client
 
+    def _get_result(
+        self,
+        *,
+        rid: str,
+        field_type: str,
+        field: str,
+        score: float,
+        score_type: ScoreType,
+    ) -> Optional[SearchResource]:
+        resource = self.client.get_resource(rid)
+        if field_type == "t":
+            text = resource.data.texts[field].value.body
+        elif field_type == "a":
+            text = resource.data.generics[field].value
+        elif field_type == "f":
+            filename = resource.data.files[field].value.file.filename
+            text = f"File: {filename}"
+        elif field_type == "l":
+            uri = resource.data.links[field].value.uri
+            text = f"Link: {uri}"
```

Comment:
Should we return the piece of text (paragraph(s)) that matched instead?
Or are we expecting the caller to do an extra GET request to get it?